test/hodo/user-create-page.test.js

```diff
@@ -0,0 +1,33 @@
+import { App } from '@bigtest/interactor';
+import test from '../helpers/base-steps/simulate-server';
+
+import Button from '../interactors/Button';
+import Checkbox from '../interactors/Checkbox';
+import Header from '../interactors/Header';
+import Link from '../interactors/Link';
+import Option from '../interactors/Option';
+import Select from '../interactors/Select';
+import Span from '../interactors/Span';
+
+export default test('user create page')
+  .step('visit "/users"', () => App.visit('/users'))
+  .step(Button.findByAttribute('data-test-pane-header-actions-button').click())
+  .step(Link('New').click())
+  .assertion(Header.findByAttribute('data-test-pane-header-title').find(Span('data-test-header-title', { value: 'Create User' })).exists())
```

Comment:
Oh the reason why is because `Header` doesn't have `textContent` or `value`.
```
<h2 data-test-pane-header-title="true" class="paneTitle---25LYc">
  <div class="appIcon---1l5fx small---3Lbqo icon-alignment-center---sQq84">
    <span class="icon---1xNd8" aria-hidden="true">
      <img src="/img/icons/app.b071ae8934eb06f25590023917b9341c.png" alt="Create, view and manage users">
    </span>
  </div>
  <span class="paneTitleLabel---2c7PC">
    <span data-test-header-title="true">Create User</span>
  </span>
</h2>
```